doc/source/developer_guide/importers_plugins.rst

```diff
@@ -0,0 +1,282 @@
+.. _importers-plugins:
+
+================================
+Create your own importers plugin
+================================
+
+Since version 3.4, pysteps allows the users to add new importers by installing external
```

Comment:
```suggestion
Since version 1.4, pysteps allows the users to add new importers by installing external
```
I suppose this feature will be included in release 1.4.

Reply:
Oops, I meant 1.4 :laughing: